tests/unit/test_asn1time.cpp

```diff
@@ -65,7 +65,7 @@ TEST_F(Asn1TimeTest, parseFromUTCTime)
 TEST_F(Asn1TimeTest, parseFromGeneralizedTime)
 {
     auto refFromTimePoint = Asn1Time::fromTimePoint(tp_2017_3_27__19_00_38);
-    auto fromGeneralizedTime = Asn1Time::fromString("20170327190038");
+    auto fromGeneralizedTime = Asn1Time::fromString("20170327190038Z");
```

Comment:
Is this a breaking change in OpenSSL? When we might want to compensate for this in our library to avoid client code adaptions